mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -85,7 +85,7 @@ def can_flash(target):
         return True
 
     # pylint: disable=too-many-return-statements, duplicate-except
-    def flash(self, source, target, method, no_reset):
+    def flash(self, source, target, method, no_reset, target_filename):
```

Comment:
what do you think, is this "public API" ? I would say no... Note that this change is breaking change if we think about this is public API..

Reply:
It should not break if default value is given for the target_filename. When the value is default it should work as before this change.

Reply:
Anyway, this is not public API IMO either.